### DISCUSSION TITLE
How to log the hovered hit in react-instantsearch-dom autocomplete?

### DISCUSSION THREAD

**Discussion by duhaime** [algolia/instantsearch#5291] [Q&A] [answered]
In [this autocomplete app](https://codesandbox.io/s/v62z0jz365), users can highlight results with arrow keys or by hovering with their mouse. 

How can one log the focused/hovered hit when the user focuses on that hit?

**Comment by Haroenv** [comment, marked as answer]
This is actually a question for https://github.com/moroshko/react-autosuggest, as that's the library you use for the autocomplete. However, you can use `onSuggestionHighlighted`, as in this example: https://codesandbox.io/s/blazing-bash-bvgpt?file=/src/AutoComplete.js:1815-1892

for more info, see those docs: https://github.com/moroshko/react-autosuggest#on-suggestion-highlighted-prop

**Reply by duhaime** [reply]
Fantastic, thank you @Haroenv!

**Reply by Haroenv** [reply]
Glad that solved what you were looking for :)
